Directory.Packages.props

```diff
@@ -5,7 +5,7 @@
 	</PropertyGroup>
 	<ItemGroup Label="Runtime Dependencies">
 		<PackageVersion Include="Funcky" Version="[3.0.0, 4)" />
-		<PackageVersion Include="Microsoft.EntityFrameworkCore" Version="[3.1.25, 8)" />
+		<PackageVersion Include="Microsoft.EntityFrameworkCore" Version="[8.0.8, 9)" />
```

Comment:
If possible, keep 3.1.25 as the minimum - does that work?

Reply:
Fixed